app/src/main/java/org/fossasia/openevent/events/ErrorDialogEvent.java

```diff
@@ -0,0 +1,59 @@
+package org.fossasia.openevent.events;
+
+import android.util.Log;
+
+import org.fossasia.openevent.OpenEventApp;
+import org.fossasia.openevent.api.APIClient;
+
+import retrofit.RetrofitError;
+
+/**
+ * Created by shivenmian on 28/12/15.
+ */
+public class ErrorDialogEvent {
+
+    public String ErrorType;
+    public String ErrorDesc;
+    public static final String TYPE = "RetrofitError Type";
+    public static final String ERROR_CODE = "Error Code";
+
+    public ErrorDialogEvent(String s1, String s2){
+        this.ErrorType = s1;
+        this.ErrorDesc = s2;
+    }
+
+    public void ErrorHandlerFunction(RetrofitError cause, ErrorDialogEvent event){
+
+        if (cause.getKind().equals(RetrofitError.Kind.CONVERSION)) {
```

Comment:
My IDE seems to be okay with this:
![image](https://cloud.githubusercontent.com/assets/547235/12030917/a98c8f5a-ae40-11e5-92b4-9f9b883d25b0.png)

>Then I took the cases as merely CONVERSION, NETWORK, HTTP etc. The code worked without errors. But then I took the base url as a malformed JSON (this) and ran. Instead of conversion error, it gave HTTP 200 error.

200 is success NOT error.